src/support/hash_city.c

```diff
@@ -306,10 +306,10 @@ static inline uint64_t CityHash64(const char *s, size_t len) {
 	if (len <= 32) {
```

Comment:
Is it ok to edit this file?

It appears to be based of Google's CityHash. It already has some WiredTiger functions in it so I figured that it's ok.

Reply:
I'd toss in a comment just so the next person knows there's no real change here (or just turn off the linter for these lines, this isn't really "our" code).

Reply:
772866d
I just decided to turn off the linter here. I can imagine that if we keep making code changes to this file it could become confusing when comparing to the original source.